_projects/armv7a9-zynq7000-zedboard/build.project

```diff
@@ -31,12 +31,12 @@ PREINIT_SCRIPT=(
 
 # Production user script contains applications to run Phoenix-RTOS
 USER_SCRIPT=(
+	"wait 500"
```

Comment:
I suppose that this change can be applied to the rest of the zynq7000 targets. In the future, there should be tests for qemu and zturn as well.